pkg/shipclient/release.go

```diff
@@ -59,26 +59,9 @@ type GraphQLResponseLintRelease struct {
 }
 
 type ShipReleaseLintData struct {
-	Messages []*ShipLintMessage `json:"lintRelease"`
+	Messages []types.LintMessage `json:"lintRelease"`
```

Comment:
was able to save a lot of code by just deserializing directly into the `types` object instead of manually copying from the `shipclient` type into the `types` type

Reply:
Nice